### PR TITLE
pamac: add page

### PR DESCRIPTION
**Pull request by Iniesta8**
Add a page for the command line utility of the GUI package management pamac which is also called pamac.

----
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/pamac.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+> Command line utility of the GUI package management pamac.
+
+- Check for package updates:
+
+`pamac checkupdates`
+
+- Install a new package:
+
+`pamac install package-name`
```
Comment (pxgamer) [inline pages/linux/pamac.md]: Change to use token syntax for this:
```suggestion
`pamac install {{package_name}}`
```

File: pages/linux/pamac.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Remove a package and its no longer required dependencies (orphans):
+
+`pamac remove -o package-name`
```
Comment (pxgamer) [inline pages/linux/pamac.md]: And here:
```suggestion
`pamac remove -o {{package_name}}`
```

File: pages/linux/pamac.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Search the package database for a package:
+
+`pamac search package-name`
```
Comment (pxgamer) [inline pages/linux/pamac.md]: And here:
```suggestion
`pamac search {{package_name}}`
```

File: pages/linux/pamac.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# pamac
+
+> Command line utility of the GUI package management pamac.
```
Comment (agnivade) [inline pages/linux/pamac.md]: `A command line utility for the GUI package manager pamac`

File: pages/linux/pamac.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Check for package updates:
```
Comment (agnivade) [inline pages/linux/pamac.md]: This can go to the end.

**Comment by Iniesta8** [comment]
All requested changes implemented.

**Pull request merged**